.github/workflows/test-iot-device-sdk-mqtt5-client.yml

```diff
@@ -0,0 +1,36 @@
+name: Java CI test-mqtt5-client
```

Comment:
`Java CI UAT`

Reply:
Fixed in d73e03879e39a7be0dc17306d2b33f6da9cff331